src/main/java/org/folio/consortia/controller/PublicationController.java

```diff
@@ -0,0 +1,25 @@
+package org.folio.consortia.controller;
+
+import java.util.UUID;
+
+import org.folio.consortia.domain.dto.PublicationRequest;
+import org.folio.consortia.domain.dto.PublicationResponse;
+import org.folio.consortia.service.PublicationService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequestMapping("/consortia/{consortiumId}")
+@RequiredArgsConstructor
+public class PublicationController implements org.folio.consortia.rest.resource.PublicationsApi {
+
+  private final PublicationService publishCoordinatorService;
+
+  @Override
+  public ResponseEntity<PublicationResponse> publishRequests(UUID consortiumId, PublicationRequest publicationRequest) {
```

Comment:
Should there be limits that would prevent a call that is not from the central tenant to work?

Reply:
In each request: to manage affiliations, publish coordinator, sharing instances and settings we have validation if a consortium exists. For member tenants the consortium table always will be empty and so consortium not found exception will be thrown.
It is our standardize pattern to validate the requests should work only in scope of central tenant